diag_manager/fms_diag_object.F90

```diff
@@ -396,17 +421,298 @@ function diag_obj_is_static (obj) result (rslt)
     rslt = obj%static
 end function diag_obj_is_static
 
-function diag_obj_get_vartype (obj) result (rslt)
-    class(fmsDiagObject_type), intent(in) :: obj
-    integer :: rslt
-    rslt = obj%vartype
-end function diag_obj_get_vartype
-
-function diag_obj_get_varname(obj) result (rslt)
-    class(fmsDiagObject_type), intent(in) :: obj
-    character(len=len(obj%varname)) :: rslt
-    rslt = obj%varname
-end function diag_obj_get_varname
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+!! Get functions
+
+!> Gets metedata
+!! @return metadata string array, or a single space if metadata is not allocated
+function get_metadata (obj) result(rslt)
```

Comment:
I **think** `result(rslt)` should be its own line ???
https://github.com/NOAA-GFDL/FMS/blob/main/CODE_STYLE.md#functions

Reply:
The style guide means that the result variable is declared on it's own line, not that a continuation is used.  I did it anyway too be consistent with your functions. I'm going to update the style guide too.